xcode/Subconscious/Shared/Components/Detail/MemoDetailDescription.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Detail.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 3/15/23.
+//
+
+import Foundation
+
+/// An enum of possible detail types.
+/// Used for the navigation stack.
+enum MemoDetailDescription: Hashable {
```

Comment:
Enum describing the different kinds of detail panel their can be.

- Editor: displayed for our editable content, whether sphere or local. Previously the only kind of detail)
- Viewer: displayed for 3p sphere content.